### PR TITLE
Added several icons to MessageIcons' lib.js

### PR DESCRIPTION
**Pull request by m-p-3**
Also added some "colors" from the official branding guidelines with a reference to the source info.

Related to #2189

### CODE REVIEW

File: apps/messageicons/lib.js [hunk 1]
```diff
@@ -5,44 +5,73 @@ exports.getImage = function(msg) {
...
+  if (s=="proton mail") return atob("GBgBAAAAAAAAAAAAQAACYAAGcAAOeAAePABeXgDebwHed4Pee/fefe/efh/ef//ef//ef//ef//ef//ef//eP//cAAAAAAAAAAAA"); // icons/protonmail.png
+  if (s=="reddit" || s=="sync pro" || s=="sync dev" || s=="boost" || s=="infinity" || s=="slide") return atob("GBgBAAAAAAAAAAYwAAX4AAh4AAgwAAgAAAgAAH4AAf+AN//sf//+fn5+PDw8HDw4Hn54H//4H//4DzzwB4HgAf+AAH4AAAAAAAAA"); // icons/reddit.png
+  if (s=="signal") return atob("GBgBAAAAAL0AAYGABH4gCf+QE//IB//gL//0b//2H//4X//6X//6X//6X//6H//4b//2L//0D//gL//ID/+QYH4gVYGAcL0AAAAA"); // icons/signal.png
+  if (s=="skype") return atob("GBgBAAAAB8AAH/8AP//AP//gf8fwfwD4fgB4fjx8fj/8Pg/8PwH8P4B8P/h8Pnx+Pjx+Hhh+HwD+D8P+B//8A//8AP/4AAPgAAAA"); // icons/skype.png
+  if (s=="slack") return atob("GBgBAAAAAOcAAeeAAeeAAeeAAGeAAAeAP+ecf+eef+e+f+e+AAAAAAAAfef+fef+eef+Oef8AeAAAeYAAeeAAeeAAeeAAOcAAAAA"); // icons/slack.png
+  if (s=="snapchat") return atob("GBgBAAAAAAAAAAAAAH4AAf+AAYGAAwDAAwDAAwDADwDwDwDwDgBwBwDgBwDgDgBwHAA4OAAcHAA4D4HwB//gAH4AAAAAAAAAAAAA"); // icons/snapchat.png
+  if (ss="starbucks") return atob("GBgBAAAAAAAAAAAAD//4D//8DADMDADMDADMDAD8DAD4DADADADADADADADADgHAB/+AA/8AAAAAAAAAP//wP//wAAAAAAAAAAAA"); // icons/cafe.png
```
Comment (myxor) [inline apps/messageicons/lib.js:58]: Here is a typo `ss=` instead of `s==`

**Comment by gfwilliams** [comment]
Wow, thanks! Please can you fix that `ss=` issue?

Are these actually all apps that definitely do send notifications? For instance I'm not sure I've ever seen notification from keep notes or VLC despite having had them installed for years?

**Comment by m-p-3** [comment]
I'll fix that ASAP. I do get Keep notes notifications once in a while, as you can attach reminders to notes, and for VLC I'm not sure but since it was requested I assumed it did.

**Comment by gfwilliams** [comment]
Thanks! I think I might have to start thinking of ways to make this faster soon then ;)

**Comment by gfwilliams** [comment]
Having said that it's still only 30ms (60ms first time) to load the icons, so it's not really a big deal at the moment :)

```
t=getTime();require("messageicons").getImage("foo");getTime()-t
```

**Pull request merged**